kolibri/utils/options.py

```diff
@@ -352,3 +352,24 @@ def update_options_file(section, key, value, KOLIBRI_HOME, ini_filename="options
             file=conf.filename
         )
     )
+
+
+empy_options_exclude_key = ["Python"]
```

Comment:
I think there's a typo? 
```suggestion
empty_options_excluded_key = ["Python"]
```